samples/unit-tests/stock-tools/bindings/demo.js

```diff
@@ -398,3 +398,142 @@ QUnit.test('Bindings general tests', function (assert) {
         qunitContainer.style.display = 'block';
     }
 });
+
+QUnit.test(
+    'Bindings on multiple axes. Checks whether a pointer action returns a proper axis (#12268).',
+    assert => {
+        const chart = Highcharts.stockChart('container', {
+                yAxis: [{
+                    height: '40%',
+                    top: '10%',
+                    id: 'topYAxis'
+                }, {
+                    height: '20%',
+                    top: '20%',
+                    id: 'bottomYAxis'
+                }],
+                series: [{
+                    data: [2, 4, 3]
+                }, {
+                    type: 'column',
+                    data: [2, 4, 3],
+                    yAxis: 1
+                }]
+            }),
+            getCoordinates = chart.pointer.getCoordinates.bind(chart.pointer),
+            getAssignedAxis = chart.navigationBindings.utils.getAssignedAxis,
+            offset = 3;
+
+        let coords,
+            coordsX,
+            coordsY;
+
+        // The yAxes overlap - y coord on both of them
+        coords = getCoordinates({
+            chartX: chart.yAxis[1].left + offset,
+            chartY: chart.yAxis[1].top + offset
+        });
+
+        coordsX = getAssignedAxis(coords, 'x');
+        coordsY = getAssignedAxis(coords, 'y');
```

Comment:
Why you split declaration and assigning of the values to those variables? Can't you do assign them right away?